.scrutinizer.yml

```diff
@@ -0,0 +1,8 @@
+build:
+    tests:
+        override:
+            -
+                command: 'phpdbg -qrr vendor/bin/phpunit --coverage-clover clover.xml'
```

Comment:
the coverage report could be send by Travis to speed up the build, see https://github.com/xsolve-pl/salesforce-client/blob/master/.travis.yml and https://github.com/xsolve-pl/salesforce-client/blob/master/.scrutinizer.yml